lib/IHP/DataSync/ihp-datasync.js

```diff
@@ -216,6 +216,13 @@ function initIHPBackend({ host }) {
 }
 
 export async function createRecord(table, record) {
+    if (typeof table !== "string") {
+        throw new Error(`Table name needs to be a string, you passed ${JSON.stringify(table)} in a call to createRecord(${JSON.stringify(table)}, ${JSON.stringify(record, null, 4)})`);
```

Comment:
`you passed ${...} as a first argument in a call to ...` would give more information. The variable name is not necessarily known to the user of the function.

Reply:
I think it's mostly clear to the user. This error only happens if there's a typo or something is undefined :)